beta/src/pages/learn/index.md

```diff
@@ -389,27 +388,33 @@ Hooks are more restrictive than regular functions. You can only call Hooks *at t
 
 ## Sharing data between components {/*sharing-data-between-components*/}
 
-In the previous example, each button had its own independent counter:
+In the previous example, each `MyButton` had its own independent `count`, and when each button was clicked, only the `count` for the button clicked changed:
 
-```js {2-4}
-- MyApp
-  - MyButton (count: 3)
-  - MyButton (count: 1)
-  - MyButton (count: 2)
-```
+<DiagramGroup>
+    <Diagram name="sharing_data_child.svg" alt="TODO: Sharing data" height={734} width={814}>
+        Before clicking, each MyButton has a count value set to zero.
+    </Diagram>
+    <Diagram name="sharing_data_child_clicked.svg" alt="TODO: Sharing data" height={734} width={814}>
+        After clicking, only one MyButton count value has updated.
+    </Diagram>
+</DiagramGroup>
 
-However, you'll often need components to *share data and always update together*.
+However, often you'll need components to *share data and always update together*.
 
-To make all buttons display the same `count` and update together, you need to move the state from the individual buttons "upwards" to the closest component containing all of them. In this example, it is `MyApp`:
+To make both `MyButton` components display the same `count` and update together, you need to move the state from the individual buttons "upwards" to the closest component containing all of them.
 
-```js {1}
-- MyApp (count: 3)
-  - MyButton
-  - MyButton
-  - MyButton
-```
+In this example, it is `MyApp`:
+
+<DiagramGroup>
+    <Diagram name="sharing_data_parent.svg" alt="TODO: Sharing data" height={770} width={820}>
```

Comment:
Does this need an alt? The caption conveys what alt would've conveyed. Maybe this should be an empty alt.

Reply:
I think they're subtly different. The alt should describe what the image contains for accessibility, where the subtitle is more of an explanation which you need to consume the diagram (either by viewing or reading the alt) to understand.